ocpmodels/datasets/ase_datasets.py

```diff
@@ -118,11 +126,17 @@ def __getitem__(self, idx):
         data_object.fid = fid
         data_object.natoms = len(atoms)
 
+        if self.key_mapping is not None:
+            for _property in self.key_mapping:
+                # catch for test data not containing labels
+                if _property in data_object:
+                    new_property = self.key_mapping[_property]
+                    if new_property not in data_object:
+                        data_object[new_property] = data_object[_property]
+                        del data_object[_property]
```

Comment:
I'm open to moving this all to its own method and using it across all our dataset classes to clean up code a bit, e.g.

```
data_object = rename_keys(data_object, self.key_mapping)
```